package.json

```diff
@@ -17,17 +17,17 @@
     "@carbon/styles": "1.x"
   },
   "dependencies": {
-    "@angular-devkit/core": "^17.3.8",
+    "@angular-devkit/core": "^18.2.12",
     "@angular-extensions/elements": "^17.1.1",
-    "@angular/animations": "^17.3.12",
+    "@angular/animations": "^18.2.13",
     "@angular/cdk": "^17.3.10",
-    "@angular/common": "^17.3.12",
-    "@angular/compiler": "^17.3.12",
-    "@angular/core": "^17.3.12",
-    "@angular/forms": "^17.3.12",
-    "@angular/platform-browser": "^17.3.12",
-    "@angular/platform-browser-dynamic": "^17.3.12",
-    "@angular/router": "^17.3.12",
+    "@angular/common": "^18.2.13",
+    "@angular/compiler": "^18.2.13",
+    "@angular/core": "^18.2.13",
+    "@angular/forms": "^18.2.13",
+    "@angular/platform-browser": "^18.2.13",
+    "@angular/platform-browser-dynamic": "^18.2.13",
+    "@angular/router": "^18.2.13",
     "@carbon/styles": "1.x",
     "@ng-select/ng-select": "^12.0.7",
```

Comment:
This should be `v13.x.x` per this [version compatibility table](https://github.com/ng-select/ng-select#versions).